config/options.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/uber/jaeger-lib/metrics"
 
 	"github.com/uber/jaeger-client-go"
+
+	otobserver "github.com/opentracing-contrib/go-observer"
```

Comment:
Ok. I tried this with the latest commits I added here. And, it shows some compatibility issues, i.e., if I initialize a new observer based on go-observer and try to assign this observer with ContribObserver() in cfg.New(), I get a compatibility error : 
                have OnStartSpan(opentracing.Span, string, opentracing.StartSpanOptions) otobserver.SpanObserver
		want OnStartSpan(opentracing.Span, string, opentracing.StartSpanOptions) (jaeger.ContribSpanObserver, bool)